public/scss/_variables.scss

```diff
@@ -4,6 +4,7 @@ $color-blue-dark: #284162;
 $color-blue-light: #335075;
 $color-banner-blue: #4B98B2;
 $color-banner-blue-light: #DFF8FD;
+$color-solid-blue: #0078FB;
```

Comment:
I know the `banner` colors are different, but can we say `color-blue-solid` instead? 
It's a weak convention, but let's keep it.